cli/engine/ansible/AnsibleConfigFileCreator.py

```diff
@@ -44,11 +44,12 @@ def update_setting(self, section, key, value, append=False):
                 self.ansible_config_file_settings[section][key] = value
 
     def process_ansible_options(self):
+        self.add_setting('defaults', 'allow_world_readable_tmpfiles', 'true')  # workaround for delegate_to with become_user
         if self.ansible_options['profile_tasks']:
             self.add_setting('defaults', 'callback_whitelist', ['profile_tasks'])
+        self.add_setting('defaults', 'forks', self.ansible_options['forks'])
```

Comment:
Just a suggestion, maybe also add an if here as we have for profile_tasks? I know it should always be provided by defaults from the cli part, but if its every re-used in a different context it will not cause unexpected issues.

Reply:
`profile_tasks` has `if` because it's boolean (`action="store_true"`) 😉